### ISSUE TITLE
Can't use async methods

### ISSUE THREAD

**Issue by visualjerk** [vue-styleguidist/vue-docgen-api#5]
First of all: Really appreciate the work you put in vue-styleguidist. I have been trying to get something similar to run for some days now and just stumbled upon your repo yesterday. Amazing work!

Right now vue-docgen-api doesn't like async methods in components. Any change you could try to include them?

Really basic example:

```
<script>
  export default {
    name: 'form',
    data: {
      params: {},
      response: false
    },
    methods: {
      async onSubmit () {
        let  res = await api.post(this.params)
        this.response = response
      }
    }
  }
</script>
```

**Comment by rafaesc92** [comment]
Thanks for the message @visualjerk , I'm really glad that the vue community uses it. It's my first package in npm.

Ok, I've published the fix under v1.1.8 of vue-styleguidist.
Please try it and let me know if it works.

**Comment by visualjerk** [comment]
Works like a charm, thank you! :)
